frontend/fingertips-frontend/components/organisms/BarChart/index.tsx

```diff
@@ -0,0 +1,98 @@
+'use client';
+
+import Highcharts from 'highcharts';
+import { HighchartsReact } from 'highcharts-react-official';
+import { HealthDataForArea } from '@/generated-sources/ft-api-client';
+
+interface ChartProps {
```

Comment:
```suggestion
interface BarChartProps {
```

For consistency with the naming of the `LineChartProps`.